src/api/controllers/bookmarkController.js

```diff
@@ -0,0 +1,96 @@
+import status from '../../helpers/constants/status.codes';
```

Comment:
@truestbyheart this is just so you can take note going forward. `ft-bookmark` isn't a great name for this branch. You have the liberty of using 2 or 3 words to describe what a branch does.

Reply:
thanks @MemunaHaruna